lang/es.js

```diff
@@ -7,6 +7,12 @@ export default {
 	"components.calendar.selected": "Seleccionado.",
 	"components.calendar.show": "Mostrar {month}",
 	"components.dialog.close": "Cerrar este cuadro de diálogo",
+	"components.filter.loading": "Loading filters",
+	"components.filter.filterCount": "{number, select, max {Más de 99} all {Todos} other {{number}}}",
```

Comment:
Pretty sure we just want this to be "99+" to save space not "More than 99".

```suggestion
	"components.filter.filterCount": "{number, select, max {99+} all {Todos} other {{number}}}",
```